### PR TITLE
Add placeholder for input answer

### LINKED ISSUE #305: time question should have a mask
image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #305

### CODE REVIEW

File: src/stories/assets/question/inputWithPlaceholder.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "http://onto.fel.cvut.cz/ontologies/form-layout/has-layout-class": "text",
+  "@type": "doc:question",
+  "http://onto.fel.cvut.cz/ontologies/form/has-template": "http://onto.fel.cvut.cz/ontologies/eccairs/aviation-3.4.0.2/a-451-qt",
+  "@id": "http://vfn.cz/ontologies/fss-form/diagnosis-date-of-first-diagnosis-of-cervical-cancer-q",
+  "http://www.w3.org/2000/01/rdf-schema#label": "Time with format hhhh:mm",
```
Comment (blcham) [inline src/stories/assets/question/inputWithPlaceholder.json:6]: @LaChope I suggest to put here also:
```
form:pattern "^(0|[1-9]\\d{0,3}):[0-5]\\d$" ;
dc:description "Enter value in hours (h) and minutes (m), i.e., examples of valid values are 0:15, 12:59, 234:00, 4567:49." ;
```

To provide complete example -- i.e. test if it works ... but will i see validation message there on keypress?
If it makes sense please add it in next PR.

Comment (LaChope) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: Unfortunately, we still have this issue that it does not work... We can not insert anything.

Comment (blcham) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: ok let's create ticket for that but **not** include in AVA

Comment (blcham) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: @LaChope can we resolve this?

Comment (LaChope) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: So you mean to resolve all issues regarding not being able to insert answers in stories, right?

Comment (LaChope) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: If so, it is now in #327

Comment (blcham) [reply src/stories/assets/question/inputWithPlaceholder.json:6]: thanks

**Pull request merged**